modin/config/envvars.py

```diff
@@ -500,14 +506,32 @@ def get(self) -> dict:
         dict
             Decoded and verified config value.
         """
+        if (
```

Comment:
What if the user uses `OmnisciLaunchParameters` class to set the parameters? Next when calling `OmnisciLaunchParameters.get()` we would not get into this branch, right?

Reply:
Seems this condition should be changed to the following:
        if cls == OmnisciLaunchParameters or (
            OmnisciLaunchParameters.varname in os.environ
            and HdkLaunchParameters.varname not in os.environ
        ):